src/targets/gpu/convolution.cpp

```diff
@@ -145,9 +144,11 @@ shape miopen_convolution::find(context& ctx, const shape& output_shape, std::vec
     if(status != miopenStatusSuccess)
         MIGRAPHX_THROW("MIOpen Convolution: get solution failed");
 
-    solution_id = solutions.front().solution_id;
+    solution_id    = solutions.front().solution_id;
+    algo           = solutions.front().algorithm;
+    workspace_size = solutions.front().workspace_size;
```

Comment:
Dont reassign this variable. Just use it below directly.

Reply:
Done.